lib/util/metricsHandler.js

```diff
@@ -95,11 +121,15 @@ const writeMetric = new promClient.Counter({
  * @property {GaugeSet} logReadOffset - Set the log read offset metric
  * @property {GaugeSet} logSize - Set the log size metric
  * @property {CounterInc} status - Increments the replication status metric
- * @property {CounterInc} dataStatus - Increments the replication status metric for data operation
- * @property {CounterInc} metadataStatus - Increments the replication status metric for metadata operation
+ * @property {CounterInc} dataReplicationStatus - Increments the replication status metric for data operation
+ * @property {CounterInc} metadataReplicationStatus - Increments the replication status metric for metadata operation
+ * @property {CounterInc} dataReplicationBytes - Increments the replication bytes metric for data operation
+ * @property {CounterInc} metadataReplicationBytes - Increments the replication bytes metric for metadata operation
+ * @property {CounterInc} sourceDataBytes - Increments the source data bytes metric
  * @property {GaugeSet} lag - Set the kafka lag metric
  * @property {CounterInc} reads - Increments the read metric
  * @property {CounterInc} writes - Increments the write metric
+ * @property {HistogramObserve} latency - Observes the latency metric
  */
 const metricsHandler = {
```

Comment:
I still don't think we should be putting this all here and will have to copy paste a little bit. Unless we switch to per process prometheus client registers.